dotCMS/src/main/java/com/dotcms/contenttype/model/type/VanityUrlContentType.java

```diff
@@ -59,40 +59,40 @@ public BaseContentType baseType() {
 
 	@Override
 	public List<Field> requiredFields() {
-		int order = 0;
+
 		Field titleField = ImmutableTextField.builder().name(TITLE_FIELD_NAME)
 				.variable(TITLE_FIELD_VAR)
 				.dataType(DataTypes.TEXT).required(Boolean.TRUE).listed(Boolean.TRUE)
 				.indexed(Boolean.TRUE)
-				.sortOrder(order++).fixed(Boolean.TRUE).searchable(Boolean.TRUE).build();
+				.sortOrder(1).fixed(Boolean.TRUE).searchable(Boolean.TRUE).build();
 		Field siteField = ImmutableCustomField.builder().name(SITE_FIELD_NAME)
 				.variable(SITE_FIELD_VAR)
 				.dataType(DataTypes.TEXT).fixed(Boolean.TRUE).indexed(Boolean.TRUE)
 				.values("$velutil.mergeTemplate('/static/content/site_selector_field_render.vtl')")
-				.required(Boolean.TRUE).sortOrder(order++).listed(Boolean.FALSE).build();
+				.required(Boolean.TRUE).sortOrder(2).listed(Boolean.FALSE).build();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 2 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)